cluster/gce/gci/configure-helper.sh

```diff
@@ -48,16 +48,32 @@ function create-dirs {
   fi
 }
 
+# Formats the given device ($1) if needed and mounts it at given mount point
+# ($2).
+function safe-format-and-mount() {
+  device=$1
+  mountpoint=$2
+
+  # Format only if the disk is not already formatted.
+  if ! tune2fs -l "${device}" ; then
```

Comment:
The old code relied on
```shell
function disk_looks_unformatted() {
  blkid ${DISK}
  if [[ $? == 0 ]]; then
    return 0
  fi

  local readonly file_type=$(file --special-files ${DISK})
  case ${file_type} in
    *filesystem*)
      return 0;;
  esac

  return 1
}
```
Why switch to tune2fs instead of keeping with blkid?